uw-research-computing/gpu-jobs.md

```diff
@@ -129,12 +129,15 @@ like to submit by using the submit file option below.
 	your jobs will run in less than 12 hours, it is advantageous to indicate that they are 
 	"short" jobs because you will be able to have more jobs running at once. 
 
-- **Request Specific GPUs or CUDA Functionality (optional)**: If your software or code requires a specific version of CUDA, a certain
-type of GPU, or has some other special requirement, you will need to add
-a "requirements" statement to your submit file that uses one of the
-attributes shown above. If you want a certain class of GPU, use `CUDACapability`:
+- **Request Specific GPUs or CUDA Functionality Using `require_gpus` (optional)**: If your software or code requires a certain
+type of GPU, or has some other special requirement, there is a special submit file line 
+to request these capabilities, `require_gpus`. Each GPU attribute that can be requested 
+in this way is structured as `GPUs_feature`. You can request specific features by using 
+the feature name in the `require_gpus` statment.  For example, if you want a certain 
+class of GPU, represented by 
+the attribute `GPUs_Capability`, your `require_gpus` statement would look like this: 
```

Comment:
Do we (and can we help the user) understand why you use 'GPUs_Capabilit' in the preceding text, but 'Capability' in the example?

Reply:
That's how TJ coded it -- that the require piece just needs the feature and not the GPU prefix. i can try to make that more clear.